web/src/utils/sort.ts

```diff
@@ -0,0 +1,9 @@
+import moment from "moment";
+
+type SortFunction<T> = (a: T, b: T) => number;
+
+export const dateSort: SortFunction<Date> = (a, b) => moment(a).diff(moment(b));
+
+export const numberSort: SortFunction<number> = (a, b) => a - b;
+
+export const stringSort: SortFunction<string> = (a, b) => a.localeCompare(b);
```

Comment:
I think we talked about this before, but these functions aren't "sorting" are they? Like numberSort is subtracting the first number from the second number, right?

Reply:
This is used like this:
```
const numbers = [4,2,6]
const numberSort = (a, b) => a - b;
const sortedNumbers = numbers.sort(numberSort) //[2, 4, 6]
```

Reply:
Ah now I see where they are being used (passing to an antd component, right?).
Since they are sort method callbacks, not actually doing the sorting, can you update the names to be xxxxSortCallback so the purpose of them is clear.

Reply:
Right, I updated the sort callback functions' names to be like this "xxxxSortCallback".
Done!